katcp/server.py

```diff
@@ -34,9 +34,11 @@
 from .sampling import format_inform_v5, format_inform_v4
 from .core import (SEC_TO_MS_FAC, MS_TO_SEC_FAC, SEC_TS_KATCP_MAJOR,
                    VERSION_CONNECT_KATCP_MAJOR, DEFAULT_KATCP_MAJOR)
-from .version import VERSION, VERSION_STR
 from .kattypes import (request, return_reply)
 
+from katversion import get_version
```

Comment:
The `get_version` function is used to extract versions from git repos, but the git repo won't be around once you install katcp.

I would rather do `from . import __version__` or something similar. You should restrict `katversion` to `setup.py` and `__init__.py` and use the explicit `__version__` elsewhere.